### PR TITLE
RoR - Assignment 1 PR

### PR DESCRIPTION
**Pull request by sasaranAndrei**
This Pull Request is made for the first assignment of Wofpack Digital RoR Internship.

Made changes include:

- Code Refactoring #1 (Remove empty space, empty lines and comments)
- Code Refactoring #2 (Use kebab-case for CSS classes instead of camelCase)
- New design elements like borders
- Price Filter using JavaScript

### CODE REVIEW

File: source/index.html [hunk 1]
```diff
@@ -0,0 +1,168 @@
...
+                    </ul>    
+                </nav>
+
+            </div>
+
+
+            <h1>Ordering food</h1>
+
+            <div class="flexContainer">
+
```
Comment (chioreandan) [inline source/index.html]: You don't need all those empty spaces

File: source/index.html [hunk 2]
```diff
@@ -0,0 +1,168 @@
+<!DOCTYPE html>
+<head>
+    <link rel="stylesheet" href="./style.css">
+    <script src="./animation.js"/>
+</head>
+
+<html>
+
+
```
Comment (chioreandan) [inline source/index.html]: remove empty space

File: source/index.html [hunk 3]
```diff
@@ -0,0 +1,168 @@
...
+<html>
+
+    
+        <header>
+
+            <div class="bgImage"></div>
+
+            <div class="topContainer">
+
+                <!--LOGO-->
```
Comment (chioreandan) [inline source/index.html]: no need for comments, it is pretty obvious what this section is

File: source/index.html [hunk 4]
```diff
@@ -0,0 +1,168 @@
...
+    <script src="./animation.js"/>
+</head>
+
+<html>
+
+    
+        <header>
+
+            <div class="bgImage"></div>
+
```
Comment (chioreandan) [inline source/index.html]: remove emtpy line

**Review by chioreandan — CHANGES_REQUESTED** [review]

**Comment by sasaranAndrei** [comment]
@chioreandan @VictorMotogna @AdrianFlorian

File: source/index.html [hunk 5]
```diff
@@ -0,0 +1,115 @@
...
+                        <li><a href="#service">SERVICE</a></li>
+                        <li><a href="#blog">BLOG</a></li>
+                        <li><a href="#contacts">CONTACTS</a></li>
+                    </ul>    
+                </nav>
+            </div>
+            <h1>Ordering food</h1>
+            <div class="flexContainer">
+                <div class="tableContainer">
+                    <!-- 80 empty space for the left part-->
```
Comment (AdrianFlorian) [inline source/index.html]: do you really need the comment here? :-)

File: source/style.css [hunk 6]
```diff
@@ -0,0 +1,252 @@
...
+    display: table;
+}
+
+.flexContainer {
+    display: flex;
+    justify-content: center;
+}
+
+.firstUnderline {   
+    /* make text invisible, but let the underline color to be white */
```
Comment (AdrianFlorian) [inline source/style.css]: same here, do you really need the comment ? :D

**Review by AdrianFlorian — CHANGES_REQUESTED** [review]
Getting there, Sasa! Please add a descriptive title to the PR and a good description of the changes. Use this as an inspiration: cristichirtos/RoR-internship-assignment-1#1

Looking forward to seeing some JS :-)

**Comment by sasaranAndrei** [comment]
@chioreandan @AdrianFlorian

File: source/style.css [hunk 7]
```diff
@@ -0,0 +1,288 @@
...
+.logo-container {
+    display: flex;
+}
+
+.logo {
+    background-color: transparent;
+    position: relative;
+    width: 100px;
+    vertical-align: middle;
+
```
Comment (chioreandan) [inline source/style.css:38]: remove empty line

File: source/style.css [hunk 8]
```diff
@@ -0,0 +1,288 @@
...
+}
```
Comment (chioreandan) [inline source/style.css:39]: add empty line

File: source/style.css [hunk 9]
```diff
@@ -0,0 +1,288 @@
...
+select, .price {
+    font-weight: 600;
+}
+
+select {
+    background-color: transparent;
+    appearance: none;
+    border: none;
+    padding: 1em 1.5em;
+}
```
Comment (chioreandan) [inline source/style.css:142]: What if you need 2 add 2 selects on the same page but with the red background color?

File: source/style.css [hunk 10]
```diff
@@ -0,0 +1,288 @@
...
+    list-style-type: none;
+}
+
+h1 {
+    color: white;
+    display: flex;
+    justify-content: center;
+    font-family: 'Montserrat', sans-serif;
+    font-size: 60px;
+}
```
Comment (chioreandan) [inline source/style.css:79]: What if in another part of the page you need to have an anchor tag (<a>) but with the color red?

File: source/index.html [hunk 11]
```diff
@@ -0,0 +1,144 @@
+<!DOCTYPE html>
```
Comment (chioreandan) [inline source/index.html:1]: check tab spacing for this file. I think it is 4 spaces instead of 2

**Review by chioreandan — CHANGES_REQUESTED** [review]

File: source/animation.js [hunk 12]
```diff
@@ -0,0 +1,31 @@
...
+    });
+}
+
+function filterMenuItems(items, sliderValue){
+    Array.prototype.forEach.call(items, function(item){
+        filterMenuItem(item, sliderValue);
+    });
+}
+
+function sliderChange (){
```
Comment (AdrianFlorian) [inline source/animation.js:26]: nice, love the naming here ;)

**Review by AdrianFlorian — APPROVED** [review]
@sasaranAndrei nice job!

**Pull request merged**